edbdeploy/project.py

```diff
@@ -608,6 +609,7 @@ def deploy(self, no_install_collection):
         extra_vars=dict(
             pg_type=self.ansible_vars['pg_type'],
             pg_version=self.ansible_vars['pg_version'],
+            efm_version=float(self.ansible_vars['efm_version']),
```

Comment:
I think we shouldn't cast the value to `float` representation, for example if `efm_version` is set to `3.10` by the user, the value passed in `extra_vars` will be `3.1`, not `3.10`

Reply:
Fixed